GatherPress/gatherpress
# Translate and retrieve singular or plural form based on the supplied number of guests.

### Description of the Change
This PR polishes the output of *%d guest(s)* within several blocks to use correct singular *or* plural forms, dependening on the amount of guests. This is done using core function of [`_n`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-i18n/#n), which:

>Translates and retrieves the singular or plural form based on the supplied number.





### How to test the Change


1.  Add, publish and view published Event
2. RSVP some Guests
3. See numbers counting and strings swapped between „1 guest“ and „2 guests)“

### Changelog Entry

Changed - Use singular and plural form strings in Event-List- and RSVP-Block.

### Credits

Props @carstingaxion

### Checklist:
